.github/workflows/connector-version-radar.yml

```
@@ -154,8 +156,13 @@ jobs:
   live:
     name: Upgrade ${{ matrix.connector }} ${{ inputs.baseline_version || matrix.baseline_version }} to ${{ inputs.candidate_version || matrix.candidate_version }}
     needs: detect
-    if: ${{ needs.detect.outputs.any_new == 'true' && github.ref == 'refs/heads/main' }}
-    runs-on: [self-hosted, macOS, ARM64, connector-lab, defenseclaw-macos]
+    # Candidate releases are untrusted code. CONNECTOR_RADAR_LIVE_ENABLED must
+    # remain unset/false until admins provision the dedicated ephemeral label
+    # with an isolated OS account, HOME/Keychain, scoped credentials, and
+    # teardown-after-one-job semantics. The persistent detector runner is not
+    # an acceptable fallback for this job.
+    if: ${{ needs.detect.outputs.any_new == 'true' && github.ref == 'refs/heads/main' && vars.CONNECTOR_RADAR_LIVE_ENABLED == 'true' }}
+    runs-on: [self-hosted, macOS, ARM64, connector-lab-ephemeral, defenseclaw-macos]
```

actionlint (1.7.12)

[error] label "connector-lab-ephemeral" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)